index.php

```diff
@@ -4,9 +4,89 @@
 <html lang="en">
 <head>
     <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
     <title>Totally Pawesome</title>
+    <link rel="stylesheet" type="text/css" href="src/assets/css/vendor/bootstrap.min.css">
+    <link rel="stylesheet" type="text/css" href="src/assets/css/styles.css">
 </head>
-<body>
 
+<body>
+<section class="row banner">
+    <div class="home col-xs-2">
+        <a href="#">
+            <h2>Home</h2>
+        </a>
+    </div>
+    <div class="shop-name col-xs-10">
+        <h2>Totally Pawesome</h2>
+    </div>
+</section>
+<section class="row main">
+    <div class="sidebar col-xs-2">
```

Comment:
Row responsiveness seems to have changed, the side bar is not large enough in smaller sizes.

Reply:
- to add, it may need a minimum width